vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2793,30 +2804,55 @@ private void onRoomTyping() {
     /**
      * Refresh the collapsed or the expanded headers
      */
-    private void updateActionBarTitleAndTopic() {
+    private void updateActionBarTitleAndSubTitle() {
         setTitle();
-        setTopic();
+        updateSubTitle();
     }
 
     /**
      * Set the topic
      */
-    private void setTopic() {
-        String topic = null;
+    private void updateSubTitle() {
         if (null != mRoom) {
+            mActionBarInfo.setVisibility(View.VISIBLE);
+            mActionBarTopic.setVisibility(View.GONE);
+
+            String roomAccessRule = DinsicUtils.getRoomAccessRule(mRoom);
+            if (mRoom.isEncrypted()) {
+                if (TextUtils.equals(roomAccessRule, RoomAccessRulesKt.RESTRICTED)) {
+                    mActionBarTypeInfo.setText(R.string.tchap_room_long_private_room_type);
+                    mActionBarTypeInfo.setTextColor(ContextCompat.getColor(this, R.color.tchap_coral_color));
```

Comment:
It's possible to use a Spannablestring instead of multiple textviews.
https://medium.com/androiddevelopers/spantastic-text-styling-with-spans-17b0c16b4568